.github/CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ get involved:
   from the `master` branch and create a 
   [pull request](https://help.github.com/articles/about-pull-requests) for your change.
   Note that new notebooks should be placed in the relevant part of the 
-  [Community](./community/) section.
+  [Community](https://github.com/qiskit-community) section.
```

Comment:
This isn't really a section. And the preceding text doesn't make sense with this.

Shouldn't this whole section just direct people to contribute to https://github.com/qiskit-community/qiskit-community-tutorials?

Reply:
Yes, you are right. There was no section called **Community**. I assumed it meant an external link which would be probably [Qiskit-Community](https://github.com/qiskit-community). I will update the link.